base_layer/wallet/src/output_manager_service/recovery/standard_outputs_recoverer.rs

```diff
@@ -265,37 +274,26 @@ where
     /// seen so far.
     async fn update_outputs_script_private_key_and_update_key_manager_index(
         &mut self,
-        output: &mut WalletOutput,
+        output: &WalletOutput,
```

Comment:
Good catch; the wallet output already has the correct info.

Reply:
this function is anyway adding in the wrong info